src/hmac.rs

```diff
@@ -79,3 +82,29 @@ where
     let keypair = K::from(secret_key);
     Ok(keypair)
 }
+
+////////////////////////////////////////////////////////////////////////
+/// HMAC-SHA256 based functions
+
+pub fn hmac(key: &[u8], message: &[u8]) -> Digest {
```

Comment:
Although this is fine for now, HMAC key is not different than an AES key (re do we handle it as `[u8]` or we create a specific struct called `HmacKey` as we do in AES. I'd suggest you sync up with @jonas-lj to have a uniform solution for these.

Reply:
Introduced HmacKey for this